### DISCUSSION TITLE
How to measure the external torque exerted on a passive rotating joint in a closed chain?

### DISCUSSION THREAD

**Discussion by HuNingHe** [cyberbotics/webots#4464] [Q&A]
I try to create a motor device on this passive hinge joint, but this results in the passive joint generating an active torque to counteract the external force.
I want to know how the torque on the passive joint changes with the active joint movement
Is there any possible way？Thanks

**Comment by omichel** [comment]
Yes, depending on how the motor was configured, it will try to keep a position and hence will apply a torque to maintain this position. If you want to disable the torque resulting from this, you can set it to 0 with [wb_motor_set_available_torque](https://www.cyberbotics.com/doc/reference/motor#wb_motor_set_available_torque), so that it will disable the position servo control.
